coralibre-sdk/testapp/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Testapp</string>
```

Comment:
I know the test app does not need to be refined at all, but I would prefer having an identifiable name like "CoraLibre-android-sdk testapp"

Reply:
done